cloudbrain/rt_server/rt_server.py

```diff
@@ -118,6 +129,10 @@ def on_close(self):
     def send_heartbeat(self):
         self.broadcast(self.clients, 'message')
 
+    def metric_exists(self, device_id, device_name, metric):
```

Comment:
It's debatable, but I think it would be nicer if this method was static. You can pass it self.subscribers. Take it out of the class and put it at the top the file along with `recursivedict`.

```
def metric_exists(subscriber, device_id, device_name, metric):
   # do stuff
   # return True / False
```